packages/catalog-process/test/db.test.ts

```diff
@@ -68,14 +117,1079 @@ describe("database test", async () => {
+            technology: "REST",
+          },
+          getMockAuthData(mockEService.producerId)
+        );
+
+        expect(id).toBeDefined();
+        const writtenEvent = await readLastEventByStreamId(id);
+        expect(writtenEvent.stream_id).toBe(id);
+        expect(writtenEvent.version).toBe("0");
+        expect(writtenEvent.type).toBe("EServiceAdded");
+        const writtenPayload = decodeProtobufPayload({
+          messageType: EServiceAddedV1,
+          payload: writtenEvent.data,
+        });
+
+        const eService: EService = {
+          ...mockEService,
+          createdAt: new Date(Number(writtenPayload.eService?.createdAt)),
+          id,
+        };
+
+        expect(writtenPayload.eService).toEqual(toEServiceV1(eService));
+      });
+
+      it("should throw eServiceDuplicate if the eService already exists", async () => {
+        await addOneEService(mockEService);
+        expect(
+          catalogService.createEService(
+            {
+              name: mockEService.name,
+              description: mockEService.description,
+              technology: "REST",
+            },
+            getMockAuthData(mockEService.producerId)
+          )
+        ).rejects.toThrowError(eServiceDuplicate(mockEService.name));
+      });
+    });
+
+    describe("update eService", () => {
+      it("should write on event-store for the update of an eService", async () => {
+        const updatedName = "eService new name";
+        await addOneEService(mockEService);
+        await catalogService.updateEService(
+          mockEService.id,
+          {
+            name: updatedName,
+            description: mockEService.description,
+            technology: "REST",
+          },
+          getMockAuthData(mockEService.producerId)
+        );
+
+        const updatedEService = {
+          ...mockEService,
+          name: updatedName,
+        };
+
+        const writtenEvent = await readLastEventByStreamId(mockEService.id);
+        expect(writtenEvent.stream_id).toBe(mockEService.id);
+        expect(writtenEvent.version).toBe("1");
+        expect(writtenEvent.type).toBe("EServiceUpdated");
+        const writtenPayload = decodeProtobufPayload({
+          messageType: EServiceUpdatedV1,
+          payload: writtenEvent.data,
+        });
+
+        expect(writtenPayload.eService).toEqual(toEServiceV1(updatedEService));
+      });
+
+      it("should throw eServiceNotFound if the eService doesn't exist", async () => {
+        expect(
+          catalogService.updateEService(
+            mockEService.id,
+            {
+              name: "eService new name",
+              description: "eService description",
+              technology: "REST",
+            },
+            getMockAuthData(mockEService.producerId)
+          )
+        ).rejects.toThrowError(eServiceNotFound(mockEService.id));
+      });
+
+      it("should throw operationForbidden if the requester is not allowed", async () => {
```

Comment:
this authorization test refers only to the call made by another organization.
Operation forbidden in returned also in case of same org but wrong role, so I would suggest to be more explicit in the test description, something like
```suggestion
      it("should throw operationForbidden if the requester is not the producer", async () => {
```

same applies for all other similar tests in the file